public/js/components/app.js

```diff
@@ -81,14 +81,7 @@ export class App extends Component {
     };
 
     this._getTmsSource = async (tmsLayerConfig) => {
```

Comment:
```suggestion
     this._getTmsSource = (cfg) => cfg.getVectorStyleSheet();
```
and delete two subsequent lines.
The `async () => { return await ... }` construct is redundant and runs a bit slower.